simapp/genesis_account.go

```diff
@@ -31,9 +31,6 @@ type SimGenesisAccount struct {
 // Validate checks for errors on the vesting and module account parameters
 func (sga SimGenesisAccount) Validate() error {
 	if !sga.OriginalVesting.IsZero() {
-		if sga.OriginalVesting.IsAnyGT(sga.Coins) {
```

Comment:
note to self/reviewer: This check needs to be handled in InitGenesis somewhere...